data/deprecated.json

```diff
@@ -1058,6 +1058,9 @@
     "old": {"office": "consultant"},
     "replace": {"office": "consulting"}
   },
+   {
+    "old": {"office": "movers"},
+    "replace": {"office": "moving_company"}
```

Comment:
Forgot to add `},` here.

Reply:
A comma on L1063 would in fact be invalid JSON, this is fine.